Assets/Python/Screens/PlatyBuilder/CvPlatyBuilderScreen.py

```diff
@@ -514,6 +514,9 @@ def placeObject( self, bMulti=False ) :
 			self.m_pCurrentPlot.setImprovementType(self.iSelection)
 		elif self.iPlayerAddMode == "Bonus":
 			self.m_pCurrentPlot.setBonusType(self.iSelection)
+		elif self.iPlayerAddMode == "BonusVariety":
+			if self.m_pCurrentPlot.getBonusType(-1) != -1:
```

Comment:
I'm wondering if it might be a better idea to limit this on the DLL end already.

Reply:
Done.